package.json

```diff
@@ -65,6 +65,7 @@
 		"storybook:deploy": "rimraf ./storybook/dist/* && npm run storybook:build && gh-pages -d ./storybook/dist",
 		"test": "wp-scripts test-unit-js --config tests/js/jest.config.json",
 		"test:e2e": "npm run wp-env:config && cross-env NODE_CONFIG_DIR=tests/e2e/config wp-scripts test-e2e --config tests/e2e/config/jest.config.js",
+		"test:e2e-single": "npm run wp-env:config && cross-env NODE_CONFIG_DIR=tests/e2e/config wp-scripts test-e2e --config tests/e2e/config/jest.config.js -t '<NAME OF YOUR TEST>' --watch --puppeteer-interactive",
```

Comment:
I'm uncertain if we need this command, as we can easily run an individual test by specifying the test path. E.g. `npm run test:e2e-single Tax` could also be run with `npm run test:e2e-dev-watch -- tests/e2e/specs/shopper/tax.test.js`. 

I'd rather use the current commands and pipe the test URL into these commands, when required. Personally, I'd only introduce a new command if the desired action cannot be achieved with the existing commands. What are your thoughts on this?

Reply:
Yeah you're right here, I didn't realise you could pass params to the underlying jest script through the current npm script. I'll remove this and use your way of doing it, much neater, thanks 👍